cmd/plural/terraform.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pluralsh/plural/pkg/terraform"
+	"github.com/pluralsh/plural/pkg/utils"
+	"github.com/urfave/cli"
+)
+
+func terraformCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:   "init",
```

Comment:
could we rename this to install?  Since it could be confounded with `terraform init` which does something a bit different